### ISSUE TITLE
Same thing

### ISSUE THREAD

**Issue by NoxynoJoestar** [ange-yaghi/engine-sim#240]
When I launch the game a system error launches and says to me: "Could not execute the code because VBRUNTIME140_1.dll was not found. Reinstalling the program may fix this problem." I read the FAQ and I found my error I installed Visual C++ and I restarted my computer and reinstalled the "game". I tried to launch it and it gives me the same error. (I ARLEADY TRY THE SOLUTION IN THE FAQ AND IT DIDN'T WORK)

**Comment by NoxynoJoestar** [comment]
pls help

**Comment by adrix5521** [comment]
I think it's a dll file missing from your system. Download it [here](https://www.dll-files.com/vcruntime140_1.dll.html) and paste it into C:\Windows\System32\

**Comment by NoxynoJoestar** [comment]
I try but it say that I'm not allowed, and should I extract the zip file ?

**Comment by adrix5521** [comment]
You should extract the zip file. Then paste the dll into C:\Windows\System32\

If you're not allowed to paste it, you can try to log in to admin account on your PC.

**Comment by Jckf** [comment]
Does it really say vbruntime, or do you mean vcruntime?

**Comment by adrix5521** [comment]
> Does it really say vbruntime, or do you mean vcruntime?

I don't think "vbruntime" exists, so he probably meant vcruntime.

**Comment by Jckf** [comment]
> > Does it really say vbruntime, or do you mean vcruntime?
> 
> I don't think "vbruntime" exists, so he probably meant vcruntime.

I don't think so either, but it's the second time he's written exactly that, so I'm just making sure. Anyway, this file is included with the Visual C++ runtime that is linked in the article about missing DLL files on the wiki. I'm not sure what you're doing to still get that error message. Try searching for the file on your system to verify that it was actually installed.

**Comment by Neozk2008** [comment]
bonjour quand j'essaie d'ouvrir le logiciel sa me met erreur vulkan-1.dll na pas était trouve

**Comment by Jckf** [comment]
> bonjour quand j'essaie d'ouvrir le logiciel sa me met erreur vulkan-1.dll na pas était trouve

Please read [the FAQ](https://github.com/ange-yaghi/engine-sim/wiki/Frequently-Asked-Questions).

**Comment by TheoCGaming** [comment]
> > Does it really say vbruntime, or do you mean vcruntime?
> 
> I don't think "vbruntime" exists, so he probably meant vcruntime.

I wonder if it's a typo or memory corruption of some kind
